templates/contribute/style.md

```diff
@@ -94,6 +94,8 @@ See our [documentation requirements](doc.html) for more suggestions and examples
 
 ### Structuring definitions and theorems ###
 
+These guidelines hold for declarations starting with `def`, `lemma` and `theorem`. For "theorem statement", also read "type declaration of a definition" and for "proof" also read "definition body".
```

Comment:
Anyone know a good way to express "type declaration of a definition"?

Reply:
"type of a definition"? "declared type of a definition"?

Reply:
```suggestion
These guidelines hold for declarations starting with `def`, `lemma` and `theorem`. For "theorem statement", also read "type of a definition" and for "proof" also read "definition body".
```